data/json/eocs/door_locking.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_LOCK_THE_DOOR",
+    "condition": { "not": "u_is_outside" },
+    "effect": [
+      { "u_location_variable": { "global_val": "door_to_close" }, "min_radius": 1, "max_radius": 1 },
+      {
+        "u_transform_radius": 1,
+        "ter_furn_transform": "lock_the_door",
+        "target_var": { "global_val": "door_to_close" }
+      },
+      { "u_assign_activity": "ACT_GENERIC_EOC", "duration": "2 seconds" },
+      { "u_message": "You rotate the deadbolt lock, and close the door from the inside." }
+    ],
+    "false_effect": { "u_message": "Can't close the door from the inside." }
```

Comment:
```suggestion
    "false_effect": { "u_message": "Can't lock this door from the outside." }
```

Reply:
~~that's why you don't make a prs at 1 am~~